build_msvc/test_bitcoin/test_bitcoin.vcxproj

```diff
@@ -14,10 +14,8 @@
     <ClCompile Include="..\..\src\test\gen\*_gen.cpp" />
     <ClCompile Include="..\..\src\wallet\test\*_tests.cpp" />
```

Comment:
Can this list be sorted? (e.g. the vim editor has a `sort` command)

This makes it easier to spot duplicates

Reply:
Sure, good idea. Done.